common/app/views/support/Commercial.scala

```diff
@@ -197,36 +205,66 @@ object Commercial {
 
   object TrackingCodeBuilder extends implicits.Requests {
 
-    def mkInteractionTrackingCode(frontId: String,
-                                  containerIndex: Int,
-                                  container: ContainerModel,
-                                  card: CardContent)(implicit request: RequestHeader): String = {
+    private case class TrackingCode(
+      context: String,
+      edition: Edition,
+      frontId: String,
+      containerIndex: Int,
+      containerTitle: String,
+      sponsorName: String,
+      cardIndex: Int,
+      cardHeadline: String
+    ) {
+      val pipeSeparated =
+        s"$context | ${ edition.id } | $frontId | ${ containerIndex + 1 } | $containerTitle | $sponsorName | " +
```

Comment:
Yes they go through to GA as '&' and '"' respectively and HTML is valid so I take that as a success.